cluster/images/hyperkube/Makefile

```diff
@@ -21,6 +21,7 @@ REGISTRY?=gcr.io/google-containers
 ARCH?=amd64
 HYPERKUBE_BIN?=_output/dockerized/bin/linux/$(ARCH)/hyperkube
 
+# TODO(dixudx): update the tag to 0.2 when the image get pushed
```

Comment:
@luxas The CI will get failed since `gcr.io/google-containers/debian-hyperkube-base-amd64:0.2` has not been pushed to upstream. I've tried before.